lib/breathe_client.rb

```diff
@@ -82,23 +82,32 @@ def sickness_events(person:, after:)
     def training_events(person:, after:)
       events = trainings(employee_id: person.breathe_id, after: after)
         .map { |training|
-          start_date = training[:start_date]&.to_date
+          start_date = training[:start_on]&.to_date
 
-          next if start_date.nil?
+          p training.to_h
```

Comment:
Probably don't want to leave this in here.

Reply:
Hah, I only added it because it was from your patch via Slack ;)

Reply:
Sorry 🤭